test/script_tools.py

```diff
@@ -43,6 +41,16 @@ class TestExtendedSimpleOption(TestCase):
     """
     Tests for the ExtendedSimpleOption class.
     """
+    def setUp(self):
+        """Backup sys.argv"""
+        super(TestCase, self).setUp()
+        self._old_argv = sys.argv
```

Comment:
no, the reference tot sys.argv is overwritten in the next line. (the [1:] is a copy there.)
But I can make it a bit more explicit for readability if this is confusing
```
>>> import sys
>>> sys.argv = ['foo']
>>> a = sys.argv
>>> sys.argv = sys.argv[:1]
>>> a
['foo']
>>> sys.argv
['foo']
>>> sys.argv.append('bar')
>>> a
['foo']
>>> sys.argv
['foo', 'bar']
>>> 
```